src/js/common/components/form-elements/ErrorableTextarea.jsx

```diff
@@ -100,6 +101,7 @@ class ErrorableTextarea extends React.Component {
             disabled={this.props.disabled}
             className={classes}
             aria-describedby={errorSpanId}
+            aria-labelledby={`${this.inputId}-label`}
```

Comment:
Originally added `aria-label`, but it seems redundant and wrong to have it if the preceding `label` is visible. Instead of setting `#msg-reply .msg-write label { display: none; }`, I set it to `left: -9999em` so that it can still be read by screen readers, and appropriately set `aria-labelledby` on the textarea.